tests/lib/linter/linter.js

```diff
@@ -13381,21 +13381,22 @@ describe("Linter with FlatConfigArray", () => {
 									assert.strictEqual(messages.length, 2);
 									assert.strictEqual(
 										messages[0].ruleId,
-										"test/my-rule",
+										null,
 									);
-									assert.strictEqual(messages[0].severity, 1);
+									assert.strictEqual(messages[0].severity, 2);
 									assert.strictEqual(
 										messages[0].message,
-										expectedRuleMessage,
+										"Unused inline config ('test/my-rule' is already configured to 'warn').",
 									);
+
```

Comment:
I'm wondering if this change should be tagged as `fix:` instead of `refactor:` since it changes the order of messages in a lint report.

Reply:
I'll mark the PR as a fix.